application/values.yaml

```diff
@@ -148,54 +177,105 @@ service:
   #   protocol: TCP
   #   targetPort: 8081
 
+# Ingress object for exposing services
 ingress:
   enabled: false
+  
+  # Port of the service that serves pods
   servicePort: 8080
```

Comment:
This should be equal to port name i.e. http so that user should change at one place only